IDEAS/Buildings/Components/Interfaces/PartialZone.mo

```diff
@@ -3,6 +3,10 @@ partial model PartialZone "Partial model for thermal building zones"
   replaceable package Medium =
     Modelica.Media.Interfaces.PartialMedium "Medium in the component"
       annotation (choicesAllMatching = true);
+
+  parameter Modelica.Fluid.Types.Dynamics energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial
```

Comment:
You are right, but it is unpratical to access that parameter if it is not propagated --> reason why I added to the zone.